doc/sphinx/source/frameworks/generic-configs.rst

```diff
@@ -128,7 +175,7 @@ Convolution shapes
 ``OMP_NUM_THREADS``
 ^^^^^^^^^^^^^^^^^^^
 
-The best resource for this configuration option is the `gnu.org site`_ 
+The best resource for this configuration option is the `gnu.org site`_.
```

Comment:
I think we should link to the Intel OpenMP docs instead, since (if I am not mistaken) that is what MKLDNN is going to use.

Reply:
Done